qiskit_nature/transformers/second_quantization/electronic/seniority_zero_transformer.py

```diff
@@ -0,0 +1,242 @@
+        self, h_1: np.ndarray, h_2: np.ndarray
+    ) -> Tuple[np.ndarray, np.ndarray]:
+        """
+        Transform PyScf integrals to the restricted formalism
+        arxiv: 2002.00035, Appendix A, Equations (A10),(A11),(A12)
+        https://arxiv.org/pdf/2002.00035.pdf
+
+        Args:
+            h_1 (np.ndarray): one-body integrals, (N x N)
+            h_2 (np.ndarray): two-body integrals, (N x N x N x N)
+
+        Returns:
+            hr_1 (np.ndarray): restricted one-body integrals, (N x N)
+            hr_2 (np.ndarray): restricted two-body integrals, (N x N)
+        """
+        # change of notation (may only be valid for PySCF -> restricted formalism)
+        h_2 = np.einsum("ijkl->ljik", h_2)
+
+        hr_1 = np.zeros((self._num_modes, self._num_modes))
+        hr_2 = np.zeros((self._num_modes, self._num_modes))
+
+        for i in range(self._num_modes):
+            hr_1[i, i] = 2 * h_1[i, i] + h_2[i, i, i, i]
+
+            for j in range(self._num_modes):
+                if i != j:
+                    hr_1[i, j] = h_2[i, i, j, j]
+                    hr_2[i, j] = 2 * h_2[i, j, j, i] - h_2[i, j, i, j]
+
+        return hr_1, hr_2
+
+
+def second_q_ops_electronic_energy(self: ElectronicEnergy) -> ListOrDictType[VibrationalOp]:
+    """Creates the second quantization operators in the restricted formalism
+    arxiv: 2002.00035, Equation (2)
+    https://arxiv.org/pdf/2002.00035.pdf
+    """
+
+    hr_1 = self.get_electronic_integral(ElectronicBasis.MO, 1).get_matrix()
+    hr_2 = self.get_electronic_integral(ElectronicBasis.MO, 2).get_matrix()
+
+    num_modes = hr_1.shape[0]
+
+    label_m = lambda x: "I" * x + "-" + "I" * (num_modes - x - 1)
+    label_p = lambda x: "I" * x + "+" + "I" * (num_modes - x - 1)
+
+    # Create pair creation and annihilation operators for the spatial orbitals
+    b = [VibrationalOp(label_m(i), num_modes=num_modes, num_modals=1) for i in range(num_modes)]
+    b_dag = [VibrationalOp(label_p(i), num_modes=num_modes, num_modals=1) for i in range(num_modes)]
+
+    # Build the operators
+    op = VibrationalOp(("I" * num_modes, 0.0), num_modes=num_modes, num_modals=1)
+    for i in range(num_modes):
+        for j in range(num_modes):
+            op += b_dag[i] @ b[j] * hr_1[i, j]
+
+            if i != j:
+                op += b_dag[i] @ b[i] @ b_dag[j] @ b[j] * hr_2[i, j]
+    op = op.reduce()
+
+    if not settings.dict_aux_operators:
+        return [op]
+
+    return {self.name: op}
+
+
+def second_q_ops_particle_number(self) -> ListOrDictType[VibrationalOp]:
+    """Returns the second quantized particle number operator.
+    The actual return-type is determined by `qiskit_nature.settings.dict_aux_operators`.
+
+    Returns:
+        A `list` or `dict` of `VibrationalOp` objects.
+    """
+    label = lambda x: "I" * x + "N" + "I" * (self._num_spin_orbitals // 2 - x - 1)
+
+    op = VibrationalOp(
+        [(label(o), 2.0) for o in range(self._num_spin_orbitals // 2)],
```

Comment:
```suggestion
    op = VibrationalOp(
        [(f"N_{o}", 2.0) for o in range(self._num_spin_orbitals // 2)],
```

Reply:
Only `+` and `-` are allowed in the sparse label of a `VibrationalOp`. However, I can use the sparse labels to do the following
```python
op = VibrationalOp([(f"+_{o}*0 -_{o}*0", 2.0) for o in range(self._num_pin_orbitals // 2)], ...)
```